### DISCUSSION TITLE
TSQL not recognized when putting project on github

### DISCUSSION THREAD

**Discussion by MateusMalvezzi** [community/community#86487] [Programming Help]
### Body

Hello, I'd like the community's help with something intriguing.
I have a study project in SQL that when I went up, everything went well and added more SQL to my profile.
I did another one yesterday analyzing fictitious sales data, I put it in my profile but unfortunately it didn't count as TSQL programming.
I'd 
like help with this, if anyone has gone through something similar.

https://github.com/MateusMalvezzi/Sales_Tracking_Project >> no SQL recognition
https://github.com/MateusMalvezzi/SQL_Portfolio >> SQL-aware

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by sivakumar-mahalingam** [comment]
The difference lies in the syntax. SQL script files in the repository should have distinguished TSQL statements in it, in order to recognize it as TSQL instead of SQL. 

You can go through this [link](https://learnsql.com/blog/14-differences-sql-vs-tsql/) to understand more about the difference. And make change in your code, so that git will index it as TSQL.
